src/libraries/CCTPLib.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.21;
+
+import { IERC20 } from "forge-std/interfaces/IERC20.sol";
+
+import { IRateLimits } from "../interfaces/IRateLimits.sol";
+import { IALMProxy }   from "../interfaces/IALMProxy.sol";
+import { ICCTPLike }   from "../interfaces/CCTPInterfaces.sol";
+
+import { RateLimitHelpers } from "../RateLimitHelpers.sol";
+
+library CCTPLib {
```

Comment:
Use the same setup thats happening in the other PR, using a struct imported from the library.

```solidity
struct TransferUSDCToCCTPParams {
        IALMProxy   proxy;
        IRateLimits rateLimits;
        ICCTPLike   cctp;
        IERC20      usdc;
        bytes32     domainRateLimitId;
        bytes32     cctpRateLimitId;
        bytes32     mintRecipient;
        uint32      destinationDomain;
        uint256     usdcAmount;
    }
```

Reply:
fixed